docker_kernel/magics/tag.py

```diff
@@ -35,24 +37,25 @@ def VALID_OPTIONS() -> dict[str, FlagDict]:
         }
     
     def _execute_magic(self) -> None:
-        target = self._args[0]
+        """
+        usage:
+        %tag source_image[:tag] target_image[:tag]
+        """
+        source_image = self._args[0]
+        target_image = self._args[1]
         try:
-            name, tag = target.split(":")
+            name, tag = target_image.split(":")
         except ValueError as e:
             # If no colon is provided the default tag is used
-            if ":" not in target:
-                name = target
-                tag = None
+            if ":" not in target_image:
+                name = target_image
+                tag = 'latest'
             else:
                 raise MagicError("Error parsing arguments:\n" + 
-                                f"\t\"{target}\" is not valid: invalid reference format")
+                                f"\t\"{source_image}\" is not valid: invalid reference format")
 
-        image_id = self._get_default_flag("image", "i", self._kernel._sha1)
-        if image_id is None:
+        if source_image is None:
```

Comment:
Remove this as a whole. The presence of all required args will already be checked by the `_find_invalid_args` method of the kernel.

Reply:
Done